### ISSUE TITLE
what is jenkins password?

### ISSUE THREAD

**Issue by Jacky2** [jenkins-x/jx#607]
I install jenkins-x to my kubernetes cluster,but open the jenkins url must input username and password??
what's the username and password?

**Comment by rawlingsj** [comment]
At the end of `jx create cluster` or `jx install` in the terminal output you should see the admin password printed.

If you have lost that you can also check your local `~/.jx/jenkinsAuth.yaml` file which contains the password too.

**Comment by Jacky2** [comment]
jx install  was faild ,but the jx pod is running in my kubernetes cluster.

Thank you your help  , I find the password in this path /root/.jx/adminSecrets.yaml.

Thank you

**Comment by rawlingsj** [comment]
OK - BTW why did the install fail?  If it did not complete successfully you may run into issues.

**Comment by Jacky2** [comment]
jx install --provider=kubernetes --skip-ingress

The error information 


Cloning the Jenkins X cloud environments repo to /root/.jx/cloud-environments
Counting objects: 397, done.
Compressing objects: 100% (4/4), done.
Total 397 (delta 0), reused 2 (delta 0), pack-reused 393
Generated helm values /root/.jx/extraValues.yaml
helm repo add jenkins-x https://chartmuseum.build.cd.jenkins-x.io
"jenkins-x" has been added to your repositories
rm -rf secrets.yaml.dec
helm repo add jenkins-x https://chartmuseum.build.cd.jenkins-x.io
"jenkins-x" has been added to your repositories
helm repo update
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
...Successfully got an update from the "stable" chart repository
...Successfully got an update from the "jenkins-x" chart repository
Update Complete. ⎈ Happy Helming!⎈ 
helm install jenkins-x/jenkins-x-platform --name jenkins-x -f ./myvalues.yaml -f ./secrets.yaml --version 0.0.507 --values=/root/.jx/gitSecrets.yaml --values=/root/.jx/adminSecrets.yaml --values=/root/.jx/extraValues.yaml --namespace=jx --timeout=6000
Error: Job failed: BackoffLimitExceeded
make: *** [install] Error 1
Error: Command failed  make ARGS=--values=/root/.jx/gitSecrets.yaml --values=/root/.jx/adminSecrets.yaml --values=/root/.jx/extraValues.yaml --namespace=jx --timeout=6000 install
error: exit status 2

**Comment by jstrachan** [comment]
@Jacky2 ah were you on premise maybe? Things normally download pretty snappily on the cloud.

BTW you can specify the admin password on the CLI via `--default-admin-password`
http://jenkins-x.io/commands/jx_install/

**Comment by rawlingsj** [comment]
Ok thanks - looks like the exposecontroller job failed to create ingress rules but I'm guessing because you disabled the nginx controller with the `--skip-ingress` flag you're planning on using NodePorts or Loadbalancer types?

**Comment by Jacky2** [comment]
@jstrachan oh, Thank you very mach.  I dont know this parameter before you tell me

**Comment by Jacky2** [comment]
@rawlingsj  because I have nginx-ingress-controller  in my kubernetes cluster, I want use this ingress .  so I use --skip-ingress and  after jenkins-x install success I will config jenkins service into the nginx-ingress-controller

**Comment by rawlingsj** [comment]
Ok FWIW I think if you had provided the `--domain` flag to `jx install` then the exposecontroller job would have created the ingress rules for you that use your ingress controller.

For example:
```
jx install --skip-ingress --domain my.hostname.io
```

**Comment by Jacky2** [comment]
Oh, thank you. but jenkins-x has multiple services. How many domains do I want to set up?

**Comment by rawlingsj** [comment]
You can use one single domain that has wildcard dns setup.  Then exposecontroller will generate a hostname for each application / service using the app name, namespace and then your domain.  

e.g.
https://jenkins.jx.my.hostname.io
https://my-application.jx-staging.my.hostname.io

**Comment by Jacky2** [comment]
Oh, Thanks. I'll try it.

**Comment by LinuxSuRen** [comment]
@rawlingsj @jstrachan 
I want to collect this topic into `jx docs` as a section named 'Jenkins'.
https://jenkins-x.io/troubleshooting/

**Comment by LinuxSuRen** [comment]
@jstrachan But how can I change the admin's password?

**Comment by rawlingsj** [comment]
Good idea - FAQ updated https://jenkins-x.io/troubleshooting/faq/

**Comment by LinuxSuRen** [comment]
I suggest we organ into serval sections, like:
https://jenkins-x.io/zh/troubleshooting/
Or the faq page perhaps has too much content.

**Comment by rawlingsj** [comment]
Sounds good - fancy popping a suggestion in as a Pull Request?  The FAQ page can be found here https://github.com/jenkins-x/jx-docs/blob/master/content/troubleshooting/faq.md

**Comment by LinuxSuRen** [comment]
I already create a PR and merge into master. But it's a Chinese version.

**Comment by DineshGinwali** [comment]
I have installed jenkins-x and it generated a password as well, but when i am login in to jenkins using admin and a password generated it fails with incorrect password.

i have verified the password at

**Comment by gbrivate** [comment]
I have noticed when I get jx installed in one machine, with gke etc, for first time, everything seems ok, then when I try to setup jx in another machine but still using the same gloud, kubernetes etc, the Jenkins password is changed, I got a new one and I'm not able to login at more, with the previous one or new one, even getting the password from files mentioned above, still not working anymore, seems the password has been ruined for some reason.

This is happening all the time when I tried to setup jx in a new machine based on a existing setup mande before, any suggestion? 

btw, I do use jx install in the second machine.
